### ISSUE TITLE
English translation ? 

### ISSUE THREAD

**Issue by klsv** [ksvc/KSYLive_iOS#64]
Please, translate KSYlive_iOS docs and changelogs to English.

**Comment by fpzeng** [comment]
This work is ongoing.

**Comment by AlexanderKozhevin** [comment]
Yeah, that's weird thing to make open-source project in national languages. 
There are developers all across the world.
